packages/@okta/vuepress-theme-default/global-components/Breadcrumb.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <nav aria-label="breadcrumb">
-    <ol class="breadcrumb">
+  <nav class="breadcrumb-nav" aria-label="breadcrumb">
+    <ol class="breadcrumb Wrap">
```

Comment:
Do we have a logic over when the HTML classes are kebab-case and when they are MixedCase?

Reply:
Nope.   its a big mix across all of the Okta css